### DISCUSSION TITLE
Vamperism perk

### DISCUSSION THREAD

**Discussion by prinnygod** [shiptest-ss13/Shiptest#472] [Ideasguy]
Allow those with the vamperism perk to drink from blood bags without any negative effects.

**Comment by Ms-Mee** [comment]
Drinking directly from bloodbags is not something anyone can do currently, they already receive no ill effects from any bloodtype, and all you need to get blood from bloodbags is an IV drip.
The ability to drink blood needs a living victim, therefore a bloodflow on them, and that seems to me like a rather unnecessary feature.

**Reply by prinnygod** [reply]
I may need to test it again but I was getting ill effects such as toxin damage from anything not O+ from an IV drip.

**Reply by Ms-Mee** [reply]
Which shouldn't happen but has happened, it worked fine in my local testing but once more not serverside
And I'll have to investigate why it's doing that

**Comment by Ms-Mee** [comment]
If your ship comes equipped with blood bags it already has an IV drip, and to extract blood to them you also need one, only buying a crate comes without one, and in that case you can buy an IV drip
Only issue is you can't construct them which is quite silly.

**Reply by terg-but-ius** [reply]
> If your ship comes equipped with blood bags it already has an IV drip, and to extract blood to them you also need one, only buying a crate comes without one, and in that case you can buy an IV drip

_unless it's the spitfire but that's a skull issue on swifter's part_
Disregarding that if you really need a blood fix an IV drip can fix that, though again _can we please make it so you can make IV drips_
